samples/sp_whoIsActive/typings/ref.d.ts

```diff
@@ -0,0 +1,7 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/// <reference path='../../../src/sql/azdata.d.ts'/>
```

Comment:
Wouldn't the point of "sample" extensions be to demonstrate how an external extension might be authored? (and so would have to use the azdata package) I don't really have a strong opinion and as far as I know there isn't any documentation linking to these samples so it's probably not a big deal - but when I saw the sample folder that's what I assumed they were meant for.